.github/workflows/reusable-e2e.yml

```
@@ -228,6 +234,22 @@ jobs:
       - name: Pull down Datastore AWS environments
         run: yarn environments datastore pull
 
+      - name: Configure liveness credentials
+        uses: aws-actions/configure-aws-credentials@v1
```

```suggestion
        uses: aws-actions/configure-aws-credentials@e1e17a757e536f70e52b5a12b2e8d1d1c60e04ef # v2.0.0 https://github.com/aws-actions/configure-aws-credentials/commit/e1e17a757e536f70e52b5a12b2e8d1d1c60e04ef
```

It's a general best practice to pin versions in CI.